### PR TITLE
Wrap saturation2 and use it for Laurent polynomials

### PR DESCRIPTION
**Pull request by thofma**
@wdecker: I found out that the new saturation is not wrapped yet on the Oscar side. We need this for the Laurent polynomial ideal business, so I would like to add it in this preliminary form. Someone can improve the integration later.

@micjoswig: This will make the example from Laura run within a few seconds (without this change I could not make it finish).

### CODE REVIEW

File: src/Rings/mpoly-ideals.jl [hunk 1]
```diff
@@ -273,9 +273,19 @@ ideal(z, x*y)
 function saturation(I::MPolyIdeal{T}, J::MPolyIdeal{T}) where T
   singular_assure(I)
   singular_assure(J)
-  K, _ = Singular.saturation(I.gens.S, J.gens.S)
+  K, _ = Singular.saturation2(I.gens.S, J.gens.S)
```
Comment (lgoettgens) [inline src/Rings/mpoly-ideals.jl]:
```suggestion
  K, _ = Singular.saturation(I.gens.S, J.gens.S)
```
Due to this change `saturation` and `_saturation2` are exactly the same. I think this change here was not meant to be made.

Comment (thofma) [inline src/Rings/mpoly-ideals.jl]:
```suggestion
  K, _ = Singular.saturation(I.gens.S, J.gens.S)
```

Comment (thofma) [reply src/Rings/mpoly-ideals.jl]: Oops, thanks!

**Comment by wdecker** [comment]
@thofma: Go ahead!

Von meinem iPhone gesendet

**Pull request merged**